### ISSUE TITLE
Flutter Web: Modal dialogs and modal alert dialogs require the aria-modal property (Accessibility)

### ISSUE THREAD

**Issue by jcollins-disney** [flutter/flutter#162124]
### Use case

The WAI-ARIA property `aria-modal="true"` must be available on all modal dialogs and modal alert dialogs so users cannot interact with content outside an active dialog window.

### Proposal

Add the WAI-ARIA property `aria-modal="true"` to the list of semantics flags on the container element for modals and alert modals.

**Comment by chunhtai** [comment]
If using role dialog or alertdialog, it will automatically have aria-modal property. Let me know if that doesn't work for you
